spec/controllers/notifications_controller_spec.rb

```diff
@@ -149,17 +149,17 @@
       FactoryGirl.create(:notification, :recipient => alice, :target => @post)
       FactoryGirl.create(:notification, :recipient => alice, :target => @post)
```

Comment:
I'd be fine with just changing the block. But two different syntaxes in one block look... weird :)